docs/reference/tutorials/create-product.rst

```diff
@@ -29,24 +29,24 @@ Run the following command to create the files:
 
 .. code-block:: bash
 
-	php bin/console sonata:product:generate Bowl sonata.ecommerce_demo.product.bowl
+	php bin/console sonata:product:generate Bowl sonata.ecommerce_demo.product.bowl App
 
-The required base files will be created in ``src/Sonata/ProductBundle``.
-To finalize the installation, we have to define the missing parameters like the type itself and the related manager. These data have to be provided in ``src/Sonata/ProductBundle/Resources/config/product.yml``.
+The required base files will be created in ``src/Application/Sonata/ProductBundle``.
```

Comment:
We should follow symfony flex style, so no `Application` subfolder please

Reply:
Got that wrong, fixed it.